packages/lodestar/src/chain/chain.ts

```diff
@@ -192,19 +203,22 @@ export class BeaconChain implements IBeaconChain {
     await this.attestationProcessor.start();
     this._currentForkDigest = computeForkDigest(this.config, state.fork.currentVersion, state.genesisValidatorsRoot);
     this.emitter.on("forkVersion", this.handleForkVersionChanged);
-    this.emitter.on("clock:epoch", this.onClockEpoch);
+    this.emitter.on("clock:slot", this.onClockSlot);
     this.emitter.on("checkpoint", this.onCheckpoint);
+    this.emitter.on("forkChoice:justified", this.onForkChoiceJustified);
+    this.emitter.on("forkChoice:finalized", this.onForkChoiceFinalized);
     this.emitter.on("error:block", this.onErrorBlock);
-    await this.restoreHeadState(state, epochCtx);
```

Comment:
we don't call this intentionally? if we don't use it, we should remove.

Reply:
Yeah, the same thing happened to me that happened to you :stuck_out_tongue:. Replayed half the blocks thru the medalla chasm (eg: to 85k) then had to start all the way back over.
Just removing this line was enough to stop restoration. :+1:  for removing restoration for now. Eventually I'd like to bring it back in certain circumstances, but with a different implementation. Fork choice persistence + restoration.